extensions/3DTILES_implicit_tiling/0.0.0/example/everything.json

```diff
@@ -94,8 +94,8 @@
         "refine": "REPLACE",
         "tilingScheme": "QUADTREE",
         "geometricError": 5000,
-        "subtreeDepth": 7,
-        "maximumDepth": 21,
+        "subtreeLevels": 7,
+        "maximumLevel": 21,
```

Comment:
@ErixenCruz these are the two renaming changes of depth -> level for clarity.

`subtreeLevels` is how many distinct levels in the tree there are. So for a quadtree with 2 levels (1 + 4 nodes), `subtreeLevels = 2`.

`maximumLevel` is the 0-indexed level of the last valid level. I know this sounds weird, as "depth" is _technically_ more accurate, but  Cesium 3D Tiles starts at level = 0 so we need to follow this convention.
![image](https://user-images.githubusercontent.com/8422414/101515353-430aa180-394c-11eb-9472-3d7411d5cb3f.png)